flask_appbuilder/tests/test_api.py

```diff
@@ -3100,3 +3100,350 @@ def test_before_request(self):
         uri = "api/v1/model1beforerequest/update"
```

Comment:
I think it makes to create a new test file for this: `tests/test_security_api.py` for example

Reply:
done.